.github/workflows/build.yml

```diff
@@ -40,9 +40,9 @@ jobs:
         uses: carlosperate/arm-none-eabi-gcc-action@v1.8.1
       - name: Install Doxygen
         run: |
-          wget https://www.doxygen.nl/files/doxygen-1.9.6.linux.bin.tar.gz
-          tar xf doxygen-1.9.6.linux.bin.tar.gz -C "$HOME"
-          echo "$HOME/doxygen-1.9.6/bin" >> $GITHUB_PATH
+          wget https://netcologne.dl.sourceforge.net/project/doxygen/rel-1.8.17/doxygen-1.8.17.linux.bin.tar.gz
```

Comment:
Looking at [the URLs used by Homebrew for Doxygen 1.8.17](https://github.com/Homebrew/homebrew-core/commit/e0b3d409123fd11e5d914a1c281bae387a8aecd9), perhaps it'd be better to use the official URL of https://downloads.sourceforge.net/project/doxygen/rel-1.8.17/doxygen-1.8.17.linux.bin.tar.gz rather than a specific SourceForge mirror?